src/metrics/activity_openrank.py

```diff
@@ -21,7 +21,7 @@
 REVIEW_COMMENT_WEIGHT = 4
 PULL_MERGED_WEIGHT = 2
 
-def getRepoActivityOrOpenrank(config, type='activity'):
+def getRepoActivityOrOpenrank(config, calType='activity'):
     """_summary_
 
     Args:
```

Comment:
In annotation, the `type` need to be changed to `calType`.

Reply:
Sure, I will fix this